smoke-test/smoke-test.mjs

```diff
@@ -3,6 +3,19 @@ import { startStandaloneServer } from '@apollo/server/standalone';
 import fetch from 'make-fetch-happen';
 import assert from 'assert';
 
+await (async function validateAllImports() {
```

Comment:
not clear why this takes a different approach from the other ones with the inline function?

Reply:
we have top-level await here, which means this could've been simpler to begin with [87669bd](https://github.com/apollographql/apollo-server/pull/6618/commits/87669bd4548451bd1c82643df6a979fd656a7653)